src/common/misc.c

```diff
@@ -734,6 +739,12 @@ daos_prop_valid(daos_prop_t *prop, bool pool, bool input)
 				return false;
 			break;
 		case DAOS_PROP_PO_ACL:
+		case DAOS_PROP_CO_ACL:
```

Comment:
What does PROP_PO and PROP_CO stand for?

Reply:
Pool prop and container prop, respectively